presto-docs/src/main/sphinx/release/release-0.284.rst

```diff
@@ -0,0 +1,67 @@
+=============
+Release 0.284
+=============
+
+**Highlights**
+==============
```

Comment:
If there are no highlights please remove this section

Reply:
This still seems to be around in the latest changes :) Please remove it .

Reply:
Oops, removed